src/twisted/internet/test/test_tcp.py

```diff
@@ -122,7 +124,16 @@
 
     getLinkLocalIPv6Addresses = _win32ifaces.win32GetLinkLocalIPv6Addresses
 
+    try:
+        from twisted.internet.iocpreactor.reactor import IOCPReactor
```

Comment:
… why is this never covered? It seems like this line should definitely see coverage

Reply:
This line is covered.

The line 133 is not covered, as we only run the `-nodeps-` tests on Linux.

Reply:
Could we convince codecov not to leave this comment by adding a `# pragma: no branch` here, maybe?